app/src/main/java/csustan/edu/cs4800/warriordiningapp/RequestNotif.java

```diff
@@ -0,0 +1,22 @@
+package com.example.notification;
```

Comment:
We need to get our package names aligned. @Dan-Land can you help confirm that everything you've created for the app is using `csustan.edu.cs4800.warriordiningapp` ?